### PR TITLE
Update to new structure

### PR DESCRIPTION
**Pull request by adamaltman**

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+      - Taxes
+  - name: Misc
+    tags:
+      - Reports
+      - Search
+      - Users
+security:
+  - SecretApiKey: []
+  - JWT: []
+paths:
```
Comment (adamaltman) [inline openapi/openapi.yaml]: This is one of the key changes, team.  There are paths explicitly defined (instead of automagically defined).

Comment (RomanHotsiy) [reply openapi/openapi.yaml]: One benefit is you can name your path files whatever you want not as long as `$ref` stays in-tact. You can also group them into folders if needed.

File: .gitignore [hunk 2]
```diff
@@ -34,3 +34,7 @@ node_modules
 
 # npm
 package-lock.json
+
+
+# OpenAPI output bundles folder
+dist
```
Comment (ashkarpetin) [inline .gitignore]: no newline

File: openapi/openapi.yaml (hunk 1 above)
Comment (slavcodev) [reply openapi/openapi.yaml]: What about definitions? Should they also be manually added?

Comment (RomanHotsiy) [reply openapi/openapi.yaml]: No, you can reference them directly from the places you use them. They will be pulled out to `components/schemas` section on bundle by filename.

E.g.: https://github.com/Rebilly/RebillyAPI/pull/796/files#diff-31e92a68a16eb684efa19ce85ffcd4ebR65

**Pull request closed without merging**